src/_pytest/config/findpaths.py

```diff
@@ -113,7 +116,7 @@ def determine_setup(
     args: List[str],
     rootdir_cmd_arg: Optional[str] = None,
     config: Optional["Config"] = None,
-):
+) -> Tuple[py.path.local, Optional[str], Any]:
```

Comment:
I see there is already a `type: py.iniconfig._SectionWrapper` below, so I think we can change the `Any` here to `Union[py.iniconfig._SectionWrapper, Dict[str, Optional[str]]]`.

Reply:
This fails as follows then though with the return of `{}`:

> error: Incompatible return value type (got "Tuple[local, Optional[str], Union[_SectionWrapper, Dict[<nothing>, <nothing>]]]", expected "Tuple[local, Optional[str], Union[_SectionWrapper, Dict[str, Optional[str]]]]")

Will use just `Dict` there - we can follow up later with "empty dict only" later there (IIRC this requires py36).

Reply:
..but it fails later with:

> src/_pytest/config/__init__.py:1052: error: Item "Dict[Any, Any]" of "Union[Any, Dict[Any, Any]]" has no attribute "config"

Let's leave it for https://github.com/pytest-dev/pytest/pull/6450.

Reply:
Yeah, let's leave it then.